build.sh

```diff
@@ -18,15 +18,16 @@ ARGS=$*
 # script, and that this script resides in the repo dir!
 REPODIR=$(cd $(dirname $0); pwd)
 
-VALIDARGS="clean librmm rmm -v -g -n -s -h"
-HELP="$0 [clean] [librmm] [rmm] [-v] [-g] [-n] [-s] [-h]
+VALIDARGS="clean librmm rmm -v -g -n -s -p -h"
+HELP="$0 [clean] [librmm] [rmm] [-v] [-g] [-n] [-s] [-p] [-h]
    clean  - remove all existing build artifacts and configuration (start over)
    librmm - build and install the librmm C++ code
    rmm    - build and install the rmm Python package
    -v     - verbose build mode
    -g     - build for debug
    -n     - no install step
    -s     - statically link against cudart
+   -p     - enable per-thread default stream
```

Comment:
Can we make this consistent between rmm and cudf? I would prefer `--ptds` personally.

Reply:
Done.